ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml

```diff
@@ -0,0 +1,27 @@
+binary_build_commands: ./mage build
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
```

Comment:
There's already a stable release which we can use.

Reply:
+1

Reply:
I did change that to `stable`, but it failed: 

```
invalid configuration: releases.latest.stream: must be one of ci, nightly
```

Reply:
I've end up using:

```
releases:
  latest:
    release:
      channel: stable
      version: "4.9"
```